channels_redis/pubsub.py

```diff
@@ -443,7 +443,7 @@ def _get_aioredis_pool(self):
     async def _get_redis_conn(self):
         await self._ensure_redis()
         conn = await self._get_aioredis_pool().acquire()
-        return aioredis.Redis(conn)
+        return aioredis.ConnectionPool(conn)
```

Comment:
I'm not sure this change was necessary. Perhaps passing the pool to `aioredis.Redis` would suffice. ConnectionPool doesn't have any client methods.